### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getScreenPosition' of undefined

### ISSUE THREAD

**Issue by Cronax42** [atom-community/autocomplete-plus#190]
[Enter steps to reproduce below:]

1. Open Atom v 0.166.0
2. Error comes up in the top-left corner
(Tab completion refused to work in the previous version of Atom but now seems to be fixed again, so the issue doesn't seem to be blocking)

**Atom Version**: 0.166.0
**System**: Mac OS X 10.9.5
**Thrown From**: [autocomplete-plus](https://github.com/atom-community/autocomplete-plus) package, v1.1.0

### Stack Trace

Uncaught TypeError: Cannot read property 'getScreenPosition' of undefined

```
At /Applications/Atom.app/Contents/Resources/app/src/text-editor.js:1325

TypeError: Cannot read property 'getScreenPosition' of undefined
  at TextEditor.module.exports.TextEditor.getCursorScreenPosition (/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:1325:34)
  at AutocompleteManager.module.exports.AutocompleteManager.runAutocompletion (/Users/svanbastenbatenburg/.atom/packages/autocomplete-plus/lib/autocomplete-manager.coffee:168:39)
  at /Users/svanbastenbatenburg/.atom/packages/autocomplete-plus/lib/autocomplete-manager.coffee:1:1

```

### Commands

```
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "welcome",
      "wrap-guide",
      "color-gutter"
    ],
    "themes": [
      "seti-ui",
      "firstlight-syntax"
    ],
    "projectHome": "/Users/svanbastenbatenburg/projects/"
  }
}
```

### Installed Packages

```coffee
# User
atom-color-highlight, v3.0.2
autocomplete-paths, v0.9.1
autocomplete-plus, v1.1.0
block-travel, v0.10.0
firstlight-syntax, v0.0.0
linter, v0.9.1
linter-pep8, v0.1.0
minimap, v3.5.0
open-last-project, v0.1.1
project-palette-finder, v2.4.7
seti-ui, v0.5.1
tabs-to-spaces, v0.8.0

# Dev
No dev packages
```

/cc @atom/core

**Comment by joefitzgerald** [comment]
This is because there is no guard in `runAutocompletion` against a null editor: https://github.com/atom-community/autocomplete-plus/blob/8b0af627627a04d22ea6388dc320ab41b42ba5ce/lib/autocomplete-manager.coffee#L165

This is already fixed in a branch I'm working on, so will be fixed when #186 is merged: https://github.com/atom-community/autocomplete-plus/blob/0b8249427ad8daa5f20ca1a03446f3e171b42016/lib/autocomplete-manager.coffee#L92-L95

**Comment by michaeljcalkins** [comment]
I just opened my editor and this error came up:
```
Uncaught TypeError: Cannot read property 'getScreenPosition' of undefined
/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:1325
Hide Stack Trace
TypeError: Cannot read property 'getScreenPosition' of undefined
  at TextEditor.module.exports.TextEditor.getCursorScreenPosition (/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:1325:34)
  at AutocompleteManager.module.exports.AutocompleteManager.runAutocompletion (/Users/michaeljcalkins/.atom/packages/autocomplete-plus/lib/autocomplete-manager.coffee:168:39)
  at /Users/michaeljcalkins/.atom/packages/autocomplete-plus/lib/autocomplete-manager.coffee:1:1
The error was thrown from the autocomplete-plus package This issue has already been reported.
```
